actions/stats_action.js

```diff
@@ -0,0 +1,31 @@
+import { STATS_REQUEST_SUCCESS, STATS_REQUEST_ERROR, URL } from './types';
+import axios from 'axios';
+
+export const getStats = token => {
+    return dispatch => {
+        return axios({
+            url: URL + '/api/global/stats-data',
+            method: 'GET',
+            headers: {
+                Authorization: 'Bearer ' + token,
+                Accept: 'application/json'
+            }
+        })
+            .then(response => {
+                console.log(
+                    'get_countries',
```

Comment:
This should log get_stats, not get_countries

Reply:
I have removed the log here..
Question -- why to send token here?? Global stats is open data (API is not protected)